modules/aggs-matrix-stats/build.gradle

```diff
@@ -28,3 +29,5 @@ restResources {
     includeCore '_common', 'indices', 'cluster', 'index', 'search', 'nodes'
   }
 }
+
+integTest.enabled = false
```

Comment:
What's creating this task? Can we structure things somehow such that projects only with yaml tests simply don't have this task at all?

Reply:
It is the PluginBuildPlugin :(  I can't really remove it or conditionally apply it until we get `javaYamlTest` (and all `internalClusterTest` are correctly moved too).

I should be able to get to that after all of the yamlRestTests are converted.